ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -404,9 +452,9 @@ Outputs:
   ConnectionURI:
     Description: 'Starts a shell session in the AWS Console'
     Value: !Sub "https://${AWS::Region}.console.aws.amazon.com/systems-manager/session-manager/${LinuxInstance}?region=${AWS::Region}"
-  NotebookConnectionURI:
-    Description: 'Notebook server login page'
-    Value: !Sub "http://${LinuxInstance.PrivateIp}:8787"
   EC2ConsoleURI:
     Description: 'Check your instance status with this link to the AWS Console'
     Value: !Sub "https://console.aws.amazon.com/ec2/v2/home?region=${AWS::Region}#Instances:search=${LinuxInstance}"
+  NotebookConnectionURI:
+    Description: 'Test link for ALB routing'
+    Value: !Sub 'https://connect.scipooldev.org/${LinuxInstance}/'
```

Comment:
This value actually gets exported by the ALB template if you want to use it: `${AWS::Region}-alb-notebook-access-ConnectionURI`  

As it is, the domain of the dev and prod will be different, so if this template is meant to go to prod (I see `internalpoolvpc`) it will need to use `scipoolprod.org` for notebooks.

Reply:
Good catch, I switched to the export